romana-install/roles/romana-common/tasks/network.yml

```diff
@@ -1,4 +1,12 @@
 ---
+# Remove the virbr0 interface
```

Comment:
Let's add a comment here about why we remove it: It's an artefact of the Devstack install and we don't need bridges or anything like it.